SpatialGDK/Source/SpatialGDK/Public/EngineClasses/AbstractVirtualWorkerTranslator.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialCommonTypes.h"
+#include "SpatialConstants.h"
+
+class SPATIALGDK_API AbstractVirtualWorkerTranslator
```

Comment:
The following comment is about every `Abstract` class: since they're abstract, why not make their functions pure virtual with `= 0`?

Reply:
Haha yep, i should